pycti/connector/opencti_connector_helper.py

```diff
@@ -308,28 +308,32 @@ def run(self) -> None:
                 self.pika_connection = pika.BlockingConnection(self.pika_parameters)
                 self.channel = self.pika_connection.channel()
                 try:
+                    # confirm_delivery is only for cluster mode rabbitMQ
+                    # when not in cluster mode this line raise an exception
                     self.channel.confirm_delivery()
                 except Exception as err:  # pylint: disable=broad-except
-                    self.helper.connector_logger.warning(str(err))
+                    self.helper.connector_logger.debug(str(err))
                 self.channel.basic_qos(prefetch_count=1)
                 assert self.channel is not None
                 self.channel.basic_consume(
                     queue=self.queue_name, on_message_callback=self._process_message
                 )
                 self.channel.start_consuming()
-            except (KeyboardInterrupt, SystemExit):
```

Comment:
I couldn't go there on my tests, as I read python documentation and discussions about catching CTRL+C or similar intereuption event, it's working only in the main thead of the process and not in a thread.